syncto/headers.py

```diff
@@ -1,11 +1,31 @@
-def convert_headers(sync_raw_response, syncto_response):
+def import_headers(syncto_request, sync_request_headers=None):
+    """Convert incoming Kinto headers into Sync headers."""
+    request_headers = syncto_request.headers
+    headers = sync_request_headers or {}
+
+    if 'If-Match' in request_headers:
+        headers['X-If-Unmodified-Since'] = '%.2f' % (
+            int(request_headers['If-Match']) / 1000.0)
```

Comment:
`If-Match` is not supposed to contain quotes ? `int('"12"')` won't work, do I miss something ?